src/style.ts

```diff
@@ -86,18 +86,16 @@ export const editorWindow: CSSProperties = {
 export const editorWrapper: CSSProperties = {
   flex: "1 1 auto",
   overflow: "hidden",
-  padding: 10,
 };
 
 export const resultsWindow: CSSProperties = {
   flex: "1 1 auto",
   position: "relative",
-  overflowY: "scroll",
```

Comment:
I think this was in place for if you resized the editor window to take up most of the window, you still needed to be able to scroll the whole results window to see the buttons. Can you confirm if this is no longer required?

Reply:
Good point - although you sacrifice screen width by having this scroll-bar present. Looking at how VSCode handles this, if you make a pane too small it collapses it altogether. Perhaps leave that as a future enhancement?

Reply:
Yep, think that would involve more UI consideration. For now I'd suggest we leave the `overflowY:scroll` in so at least it's functional

Reply:
Different approach - I've added a minimum height to the splitter so that the pane never gets too small https://github.com/ScottLogic/kdb-studio-2/pull/54/commits/fe45b1ff2308e3b1edd10aa5760e3d9835efdafc

Reply:
Cool, happy with that